### DISCUSSION TITLE
How to disable font awesome in remote theme

### DISCUSSION THREAD

**Discussion by kulbhushanchand** [mmistakes/minimal-mistakes#2850] [Q&A] [answered]
I'm using `minimal-mistakes` as remote theme and website is hosted on GitHub repo.
Since I don't want to use `font awesome`, how I can disable the loading of  `font awesome` related assets to cut-down on webpage size and loading time overheads?

Thanks.

**Comment by iBug** [comment, marked as answer]
Copy `_includes/head.html` to your repo and remove the line responsible for loading Font Awesome.

<https://github.com/mmistakes/minimal-mistakes/blob/c501842db8765d9538fc787219ca4437a6570beb/_includes/head.html#L18>

**Reply by kulbhushanchand** [reply]
Thanks @iBug. This also means in future, to manually update head.html in case of changes during theme update. Having an enhancement in the theme (like a switch in `_config.yaml`) could be nice.
